tools/ktx/command_create.cpp

```diff
@@ -783,6 +794,13 @@ Create a KTX2 file from various input files.
                 wrap | reflect | clamp.
                 Defaults to clamp.</dd>
         </dl>
+        <dt>\--normalize</dt>
+        <dd>Normalize input normals to have a unit length. Only valid for
+                linear normal textures with 2 or more components. For 2-component inputs
+                2D unit normals are calculated. Do not use these 2D unit normals
+                to generate X+Y normals for --normal-mode. For 4-component inputs
```

Comment:
"with @b \--normal-mode."

Even if you don't agree with changing "for" the other changes are needed.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/977/commits/f1b94c8fd84923b76dcee2c080d9b5735827fb37